.kokoro

```diff
@@ -157,6 +157,75 @@ upload_bazel_test_artifacts() {
   find -L . -name "sponge_log.xml" -type f | copy_to_artifacts
 }
 
+run_bazel_affected() {
+  echo "Running bazel affected..."
+
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    move_derived_data_to_tmp
+  fi
+
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    bazel version
+    use_bazel.sh 0.20.0
```

Comment:
Please move the bazel version to a variable/constant. I see it's on line 238 as well.

Reply:
Will do so in a follow-up change. https://github.com/material-components/material-components-ios/issues/6061